packages/fxa-settings/src/components/LinkRememberPassword/index.stories.tsx

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import LinkRememberPassword from '.';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { MOCK_EMAIL } from './mocks';
+
+export default {
+  title: 'components/LinkRememberPassword',
+  component: LinkRememberPassword,
+} as Meta;
+
+const storyWithProps = ({ ...props }) => {
```

Comment:
We could do something like this instead, I should have looked a bit more closely here to point out but it's not a big deal since I don't think we're doing it consistently, I think this is slightly improved and good to do in future stories:

```
const storyWithProps = (props?: LinkRememberPasswordProps) {
```

And then you could call `Default` without the empty object, like `storyWithProps()`.

Edit: oh sweet, you actually did this in another SB file with `ResetPasswordProps`, noice.